public/app/features/trails/MetricScene.tsx

```diff
@@ -115,49 +119,92 @@ export class MetricActionBar extends SceneObjectBase<MetricActionBarState> {
       setBookmarked(!isBookmarked);
     };
 
+    if (!actionView) {
+      metricScene.setActionView(actionViewsDefinitions[0].value);
```

Comment:
Do we always trust that we will want index 0, or should we just say `'overview'` here?